### PR TITLE
Expand installation instructions

### PR DESCRIPTION
**Pull request by cristianonicolai**
Related: https://github.com/ansible/ansible-dev-tools/issues/37

### CODE REVIEW

**Review by shatakshiiii — APPROVED** [review]

**Review by ajinkyau — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The curated list of tools installed as part of the Ansible Development Tools inc
 
 ## Getting started
 
-To get started, please follow the [installation](installation.md) steps to get ADT setup and jump into [Getting Started](getting-started.md) guide for details.
+To get started, please follow the [installation](installation.md) steps to get ADT setup and jump into [Getting Started](getting-started.md) guide for more details.
```
Comment (ariordan-redhat) [inline docs/index.md:39]:
```suggestion
To get started, follow the [installation](installation.md) steps to get ADT setup and jump into the [Getting Started](getting-started.md) guide for more details.
```

File: docs/installation.md [hunk 2]
```diff
@@ -12,6 +12,34 @@ hide:
 
 `pip install ansible-dev-tools`
 
+Once installation is completed, visit [Getting Started](getting-started.md) guide for more details about ADT usage.
```
Comment (ariordan-redhat) [inline docs/installation.md:15]:
```suggestion
Once installation is completed, see the [Getting Started](getting-started.md) guide for more details about ADT usage.
```

File: docs/installation.md [hunk 3]
```diff
@@ -12,6 +12,34 @@ hide:
...
+
+### Latest Releases
+
+- GitHub
+  Visit the [ADT GitHub releases page](https://github.com/ansible/ansible-dev-tools/releases) to view the latest releases. Each release includes detailed release notes outlining new features, improvements, and bug fixes.
```
Comment (ariordan-redhat) [inline docs/installation.md:20]:
```suggestion
  To view the latest releases, see the [ADT GitHub releases page](https://github.com/ansible/ansible-dev-tools/releases). Each release includes detailed release notes outlining new features, improvements, and bug fixes.
```

File: docs/installation.md [hunk 4]
```diff
@@ -12,6 +12,34 @@ hide:
...
+
+- PyPI
+  The [PyPI page for ADT](https://pypi.org/project/ansible-dev-tools/) provides information on the latest stable release and allows you to download specific versions of the package.
+
+## Upgrade
+
+To upgrade ADT to the latest version, you can use the following pip command:
```
Comment (ariordan-redhat) [inline docs/installation.md:27]:
```suggestion
To upgrade ADT to the latest version, use the following pip command:
```

File: docs/installation.md [hunk 5]
```diff
@@ -12,6 +12,34 @@ hide:
...
+
+## Downgrade
+
+If needed, you can downgrade ADT to a specific version using the following pip command:
+
+`pip install ansible-dev-tools==desired-version`
+
+## Uninstallation
+
+If you need to uninstall ADT, you can use the following pip command:
```
Comment (ariordan-redhat) [inline docs/installation.md:39]:
```suggestion
If you need to uninstall ADT, use the following pip command:
```

**Comment by cristianonicolai** [comment]
@ariordan-redhat suggestions applied, thanks

**Pull request merged**